src/Features/Blockcore.Features.Notifications/AssemblyInfo.cs

```diff
@@ -0,0 +1,3 @@
+﻿using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Blockcore.Features.Notifications.Tests")]
```

Comment:
check with @sondreb he solved this on the property files I think

Reply:
Great find, I confirmed with Sondre and resolved this